lib/internal/errors.js

```diff
@@ -13,16 +13,20 @@
 const {
   ArrayIsArray,
   Error,
+  JSONStringify,
   Map,
   MathAbs,
   NumberIsInteger,
   ObjectDefineProperty,
   ObjectKeys,
+  StringPrototypeSlice,
   Symbol,
   SymbolFor,
   WeakMap,
 } = primordials;
 
+const sep = process.platform === 'win32' ? '\\' : '/';
```

Comment:
```suggestion
const sep = path.sep;
```
? or did i already ask this in another PR.

Reply:
This shouldn't be done in a backport. I've open a PR against master to make this change

https://github.com/nodejs/node/pull/32440

Reply:
We did discuss this before - there was a circular ref issue with it as errors is lower level than process. It might be working now otherwise should be possible to work around it too in the follow-up PR.

Reply:
Thanks, ack to both of you :-)